portfolio/src/main/webapp/index.html

```diff
@@ -131,6 +136,77 @@ <h5 class="card-title">Amadeo Medical</h5>
         </div>
     </section>
 
+    <section id="yosemite-separator">
```

Comment:
I love the content and the site overall.  You did a great job.  One nit:

Consider putting each distinct section (Projects, About Me, Fwego, etc.) in their own HTML file and then importing them into your main index.html -- this will make your code more modular, maintainable, and easy to read.

Reply:
Sounds good, I did the same for the navbar since it is shared among all the major pages (created a separate navbar.html file).